### PR TITLE
Add compat data for AmbientLightSensorReading

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/docs/Web/API/AmbientLightSensorReading

### CODE REVIEW

File: api/AmbientLightSensorReading.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "AmbientLightSensorReading": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AmbientLightSensorReading/AmbientLightSensorReading",
```
Comment (maboa) [inline api/AmbientLightSensorReading.json]: I've been adding a description to constructor entries - ie 
`"description": "<code>AmbientLightSensorReading()</code> constructor",`
I'm not sure how important that is, but thought I'd mention.

Comment (bunnybooboo) [reply api/AmbientLightSensorReading.json]: :heavy_check_mark:

File: api/AmbientLightSensorReading.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
```
Comment (ddbeck) [inline api/AmbientLightSensorReading.json]: I'm not sure this is actually standards track. Doesn't seem to be in [the draft linked from the MDN page](https://www.w3.org/TR/ambient-light/), at least. Maybe this and the value below should be `false` instead?

**Comment by connorshea** [comment]
This is missing the `illuminance` property https://developer.mozilla.org/en-US/docs/Web/API/AmbientLightSensorReading/illuminance

File: api/AmbientLightSensorReading.json (hunk 2 above)
Comment (Elchi3) [reply api/AmbientLightSensorReading.json]: Yeah, this whole interface is obsolete, and was part of the old spec only. See https://www.w3.org/TR/2016/WD-ambient-light-20160830/#ambient-light-sensor-reading-interface

**Comment by Elchi3** [comment]
This whole interface is gone and was only part of an early spec. I've fixed the MDN pages and opened https://github.com/mdn/browser-compat-data/pull/2420 to add illuminance to the AmbientLightSensor interface where it belongs actually.

**Pull request closed without merging**